frontend/src/libs/components/button/styles.module.scss

```diff
@@ -142,3 +161,21 @@
     transform: rotate(360deg);
   }
 }
+
+@media (width < 900px) {
+  .icon-right {
+    left: 80%;
+  }
+}
+
+@media (width < 768px) {
+  .icon-right {
+    left: 90%;
+  }
+}
+
+@media (width < 480px) {
+  .icon-right {
+    left: 85%;
+  }
+}
```

Comment:
remove  media queries from button
```suggestion
```

Reply:
@Liza-Veis without it trashbutton position will break and it will looke like this:
![Uploading image.png…]()

Reply:
we should handle it using `right` property without media-queries

Reply:
done